docs/daisy-migrating-existing-workflows.md

```diff
@@ -0,0 +1,115 @@
+# Migrating Image Creation Workflows to Daisy
+
+This page describes how to use Daisy to automate creation of custom GCE images.
+These images are usually based on an existing base image (often a fresh install
+of your OS of choice), with some extra software you have chosen to install. This
+page assumes you have an existing workflow for creating such an image, and
+describes how to use Daisy to automate this task.
+
+To begin using Daisy for your existing workflows, follow these steps:
+
+  1. Choose a base image to start with.
+  2. Enumerate all the steps you currently follow to prepare your image.
+  3. Make sure all those steps can be done from the command line.
+  4. Write a new (or edit an existing) script to run on your base image which
+     does *all* of the steps you identified in step 2.
+  5. Write a Daisy workflow which creates at least one disk (the first disk
+     should be your base image), starts an instance from that disk, runs your
+     script, waits for the script to finish, then takes an image from the disk.
+
+The rest of this page describes these steps in greater depth.
+
+## Selecting a Base Image
+
+Your base image can come from anywhere, but if you're starting with an OS that
+is available on Compute Engine, then you can use a "partial URL" to get it
+directly from Google rather than uploading your own image.
+
+Additionally, to avoid pinning your script to a specific release, you can use
+the image "family" as a partial URL, rather than naming a specific image.
+Partial URLs for image families have the following form:
+
+    projects/<project>/global/images/family/<family-name>
+
+To learn more about the `sourceImage` property and partial URLs, see the
+["Disks"
+page](https://cloud.google.com/compute/docs/reference/latest/disks#sourceImage)
+of the Compute Engine API docs.
+
+You will need to find the correct project and family name for your chosen image
+family. To do this, navigate in the Cloud Console to Compute Engine -> Images,
+then click on the name of the instance you would like to use. On this page will
+be listed the image's family. To find the project, click on "Equivalent REST"
+and find the "selfLink" field. The second part of the path is the project name.
+Note that the project name is not the name of your own project, it is the name
+of the project which contains your chosen base image. For example, the partial
+URL for the Ubuntu 16.04 LTS image family is:
+
+    projects/ubuntu-os-cloud/global/images/family/ubuntu-1604-lts
+
+You can use this partial URL in the "SourceImage" field in a "CreateDisks" Daisy
+step, or anywhere else that Daisy accepts partial URLs. By using an image
+family, you ensure that you will always run your workflow on the most up-to-date
+base image available.
+
+## Translating your Current Workflow
```

Comment:
Ditto about Translating, as that is a term we are using to translate imported images to GCE images.

Reply:
Ack